docs/installation.md

```diff
@@ -0,0 +1,136 @@
+# Installation
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
+
+## Prerequisites
+
+The project has the following [dependencies](#dependencies):
+
+* [Algolia](#algolia)\*
+* [Auth0](#Auth0)
+* [Google Cloud](#google-cloud)\*
+* [Graphcool](#graphcool)\+
+* [Mailgun](#mailgun)\*
+
+> \* _optional, but recommended_
+>
+> \+ _local deployment available_
+
+## Installing
+
+First, clone the repository
+
+```bash
+# Path: ./
+git clone https://github.com/Zenika/FAQ
+```
+
+Then, install the dependencies of the project
+
+```bash
+# Path: ./FAQ/
+npm install
+```
+
+## Configuration
+
+Before running the project, you will need to configure your environment variables from the [dependencies](#dependencies)
+
+```bash
+# Path: ./FAQ/
+cp .env.local.example .env.local
+vim .env.local
+
+cp server/.env.local.example server/.env.local
+vim server/.env.local
+```
+
+## Dependencies
+
+### Algolia
+
+[Algolia](https://www.algolia.com/) is used for the full-text search.
```

Comment:
Either drop the "the" or add "feature" at the end ("the full-text search *feature*").